src/main/java/com/github/theholywaffle/teamspeak3/SocketReader.java

```diff
@@ -116,7 +119,7 @@ public void run() {
 
 					c.feedError(line.substring("error ".length()));
 					if (c.getError().getId() != 0) {
-						TS3Query.log.severe("TS3 command error: " + c.getError());
+						log.debug("TS3 command error: {}", c.getError());
```

Comment:
I disagree strongly with this change.

This would make it that, unless the user of the API configured their logging framework to display all messages, they'd just receive `null` or `-1` as the error to some wrong TS3Api method call with no further explanation as to what has gone wrong.

This would make even the simplest mistakes a lot harder to debug for new users of our code and would probably flood our GitHub issues page with tons of issues.

We could argue whether such an error should have the `error` or the `warning` level, but `debug` certainly isn't the right way to go with how TS3Api currently handles command errors.